Hydra/ApiDocumentationBuilder.php

```diff
@@ -362,40 +363,41 @@ private function getHydraOperation(ResourceInterface $resource, OperationInterfa
      */
     private function getRange(AttributeMetadataInterface $attributeMetadata)
     {
-        if (isset($attributeMetadata->getTypes()[0])) {
-            $type = $attributeMetadata->getTypes()[0];
+        $type = $attributeMetadata->getType();
+        if (!$type) {
+            return;
```

Comment:
Why? It's better to have an undocumented property than an error don't you think?